Marlin/src/module/stepper.cpp

```diff
@@ -223,6 +227,14 @@ volatile int32_t Stepper::endstops_trigsteps[XYZ];
 volatile int32_t Stepper::count_position[NUM_AXIS] = { 0 };
 int8_t Stepper::count_direction[NUM_AXIS] = { 0, 0, 0, 0 };
 
+#if ENABLED(LASER_POWER_INLINE_TRAPEZOID)
+  bool Stepper::laser_trap_en = false;
+  uint8_t Stepper::cur_power = 0;
+  bool Stepper::cruise_set = false;
+  uint32_t Stepper::last_step_count = 0;
+  uint32_t Stepper::acc_step_count = 0;
+#endif
```

Comment:
It would be better to put all these new data members together into a `struct`. This way generic names like `last_step_count` will have proper context. The laser-related elements in `block_t` can also be grouped into a `struct`.

Reply:
Good idea; thanks